### PR TITLE
Frontend/logo button

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-319.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-319.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-319.dev.beta.nhschoices.net)

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 node_modules/
 public/components
-public/css
+css
```
Comment (st3v3nhunt) [inline .gitignore]: Why has this changed?

File: css-build [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 OUTPUT_STYLE=${1:-compressed}
 
-echo "node-sass with output-style of ${OUTPUT_STYLE}"
+echo "node-sass with osutput-style of ${OUTPUT_STYLE}"
```
Comment (st3v3nhunt) [inline css-build]: Typo introduced

File: css-build [hunk 3]
```diff
@@ -2,10 +2,10 @@
...
 
-node-sass --output-style "${OUTPUT_STYLE}" --include-path node_modules/normalize-scss/sass --include-path scss-0.3.1 scss-c2s/c2s.scss public/css/nhsuk.css;
-node-sass --output-style "${OUTPUT_STYLE}" --include-path node_modules/normalize-scss/sass --include-path scss-0.3.1 scss-c2s/c2s-print.scss public/css/print.css;
-node-sass --output-style "${OUTPUT_STYLE}" --include-path node_modules/normalize-scss/sass --include-path scss-0.3.1 scss-c2s/c2s-ie6.scss public/css/nhsuk-ie6.css;
-node-sass --output-style "${OUTPUT_STYLE}" --include-path node_modules/normalize-scss/sass --include-path scss-0.3.1 scss-c2s/c2s-ie7.scss public/css/nhsuk-ie7.css;
-node-sass --output-style "${OUTPUT_STYLE}" --include-path node_modules/normalize-scss/sass --include-path scss-0.3.1 scss-c2s/c2s-ie8.scss public/css/nhsuk-ie8.css;
+node-sass --output-style "${OUTPUT_STYLE}" --include-path node_modules/normalize-scss/sass --include-path scss-0.3.1 scss-c2s/c2s.scss public/nhsuk.css;
```
Comment (st3v3nhunt) [inline css-build]: Why is the output being written to `./public` rather than `./public/css` now?

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-319.dev.beta.nhschoices.net)

File: css-build (hunk 3 above)
Comment (ajthomascouk) [inline css-build]: See above comment

File: .gitignore (hunk 1 above)
Comment (ajthomascouk) [inline .gitignore]: I moved the compiled css into the root of `public` as this is what it is in the library and therefore the image paths are set to that. However it should have been That should have been `public/*.css`

File: css-build (hunk 2 above)
Comment (ajthomascouk) [inline css-build]: Sorted

**Comment by ajthomascouk** [comment]
@st3v3nhunt wrote:
> it would be worth adding the --include-path paths as variables to the script 

Agreed, looks much better and easier to manage.

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services succeeded (http://connecting-to-services-pr-319.dev.beta.nhschoices.net)

**Pull request merged**